sphinx_gallery/gen_rst.py

```diff
@@ -410,17 +400,23 @@ def scale_image(in_fname, out_fname, max_width, max_height):
                           generated images')
 
 
-def save_thumbnail(thumbnail_image_path, base_image_name, gallery_conf):
+def save_thumbnail(thumbnail_image_path, src_file, gallery_conf):
     """Save the thumbnail image"""
```

Comment:
In a different PR: we should update the doc of this function and add what this function return